src/tests/ftest/util/io_utilities.py

```diff
@@ -327,86 +326,6 @@ def read_array_objects(container, size, items, dkey, akey, obj, txn):
     return [item[:-1] for item in data]
 
 
-def read_during_rebuild(
```

Comment:
Any specific reason these are removed?

Reply:
The methods io_utils that have been removed have been updated with code that resides in test_utils now and this routine was not used anywhere and was replaced with test_utils,read_data_during_rebuild